stable/mongodb/templates/ingress.yaml

```diff
@@ -0,0 +1,40 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "mongodb.fullname" . -}}
+{{- $ingressPaths := .Values.ingress.paths -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
```

Comment:
Why using this variable? Isn't it simpler to use sth like:

```yaml
metadata:
  name: {{ template "mongodb.fullname" . }}
```

Reply:
Yes, just a copy paste from `helm create` command, no big deal.